activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/mock/MockFactory.java

```diff
@@ -78,10 +75,10 @@ public static ModelEntity processModel(ProjectEntity parentProject,
     }
 
     public static ModelEntity processModelWithExtensions(String name,
```

Comment:
what is name here? process id? process name? If so can change it?

Reply:
that is the name, i've used as key to avoid restructuring the whole test structure